spark-dependencies/pom.xml

```diff
@@ -509,16 +511,25 @@
 
     <profile>
       <id>spark-1.6</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
       <properties>
         <spark.version>1.6.1</spark.version>
         <py4j.version>0.9</py4j.version>
         <akka.group>com.typesafe.akka</akka.group>
         <akka.version>2.3.11</akka.version>
         <protobuf.version>2.5.0</protobuf.version>
       </properties>
+    </profile>
 
-      <dependencies>
-      </dependencies>
+    <profile>
+      <id>spark-2.0</id>
```

Comment:
I agree with @Leemoonsoo. Even though we have a plan to support Spark 2.0 in Zeppelin 0.6.0, it would be good to apply this part after Spark 2.0 releases.